include/zeda/zeda_misc.h

```diff
@@ -140,7 +140,13 @@ __EXPORT int fpeek(FILE *fp);
  * specification.
  */
 #ifndef __KERNEL__
+
+#ifdef __WINDOWS__
```

Comment:
Why do you need this? eprintf() should output to stderr regardless of the system.